### PR TITLE
Feat/save event

### PR DESCRIPTION
**Pull request by guhmerces**
This adds the feature of saving an event by its ID

Any kind of feedback is very welcome

### CODE REVIEW

File: src/eventbrite-provider.ts [hunk 1]
```diff
@@ -9,22 +11,63 @@ function EventbriteProvider(this: any, options: any) {
...
     }
   })
 
+  async function loadEvent(this: any, msg: any) {
+    const q: any = msg.q
+    const eventID: string = q.id
+
+    const event: any = await eventbriteSDK.request(`/events/${eventID}`)
+
+    if(event.id) {
```
Comment (rjrodger) [inline src/eventbrite-provider.ts:38]: As per previous PR, this is probably not needed as we can use the eventbrite id directly. Question: is it a number? in that case we should convert to a string, as that is a more universal is format

File: src/eventbrite-provider.ts [hunk 2]
```diff
@@ -9,22 +11,63 @@ function EventbriteProvider(this: any, options: any) {
...
+  }
+
+  async function saveEvent(this: any, msg: any) {
+    const ent: any = msg.ent
+    const eventID: string = ent.id
+
+    const body = JSON.stringify({
+      event: {
+        description: {
+          html: ent.description.html,
```
Comment (rjrodger) [inline src/eventbrite-provider.ts:53]: In eventbrite is the description stored as HTML? Do they provide a custom WYSIGWYG editor on their site?

File: src/eventbrite-provider.ts [hunk 3]
```diff
@@ -9,22 +11,63 @@ function EventbriteProvider(this: any, options: any) {
...
+      },
+    })
+
+    // Missing a slash at the end of the URL cause the Fetch API to not handle POST requests correctly.
+    const event: any = await eventbriteSDK.request(`/events/${eventID}/`, {
+      method:'POST',
+      body,
+    })
+
+    if(event.id) {
```
Comment (rjrodger) [inline src/eventbrite-provider.ts:64]: See above re IDs

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: src/eventbrite-provider.ts (hunk 1 above)
Comment (guhmerces) [reply src/eventbrite-provider.ts:38]: It is a string. Thanks !

File: src/eventbrite-provider.ts (hunk 2 above)
Comment (guhmerces) [reply src/eventbrite-provider.ts:48]: Yes, they provide a WYSIWYG editor. Reading the docs more carefully, I saw that event.description will be deprecated so I changed it to event.summary, according to the documentation.
However, I read there is a feature called Structured Content with which you can make multi media event descriptions based on modules. The POST request becomes a little bit more complex, but, considering the description to be summary + structured, should we provide it ? I can make a new PR providing it. Thank you !!
https://www.eventbrite.com/platform/docs/event-description

**Pull request merged**